### ISSUE TITLE
Redirection Error

### ISSUE THREAD

**Issue by algmelo** [johngodley/redirection#649]
```
Plugin: 3.0.1
WordPress: 4.9.2
PHP: 7.0.27-1+ubuntu14.04.1+deb.sury.org+1
Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36

Error: Unexpected token < in JSON at position 0
Action: redirection/v1/group GET
Code: 404 Not Found
Raw: <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>404 Not Found</title>
</head><body>
<h1>Not Found</h1>
<p>The requested URL /wp-json/redirection/v1/group was not found on this server.</p>
<hr>
<address>Apache/2.4.7 (Ubuntu) Server at jmscomunicacao.local Port 80</address>
</body></html>


Error: Unexpected token < in JSON at position 0
Action: redirection/v1/redirect?orderby=id GET
Code: 404 Not Found
Raw: <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>404 Not Found</title>
</head><body>
<h1>Not Found</h1>
<p>The requested URL /wp-json/redirection/v1/redirect was not found on this server.</p>
<hr>
<address>Apache/2.4.7 (Ubuntu) Server at jmscomunicacao.local Port 80</address>
</body></html>

```

**Comment by johngodley** [comment]
Can you try the 3.1 beta?

https://github.com/johngodley/redirection/releases/tag/3.1

**Comment by johngodley** [comment]
I've now released version 3.1. I will close this ticket for now, but please reopen if the problem persists
